components/StoryModal.tsx

```diff
@@ -0,0 +1,33 @@
+import { Modal, Input } from '@geist-ui/react';
+import { Story } from '../redux/types';
+
+import Owners from './Owners';
+import Labels from './Labels';
+
+interface StoryModalParams {
+  story: Story;
+  isOpen: boolean;
+  close: () => void;
+}
+
+const StoryModal = ({ story, isOpen, close }: StoryModalParams): JSX.Element => {
+  if (!Boolean(story)) {
```

Comment:
Was this intentional? Should it not just be `!story`?

Reply:
Yeah it's intentional, I want to be verbose in showing that the object is not a boolean right now, and i'm using the boolean cast to get true/false.